http/service.go

```diff
@@ -32,7 +32,7 @@ const (
 // Start an intance using a new Service
 // Note that for CloudEvent Handlers this effectively accepts ANY because
 // the actual type of the handler function is determined later.
-func Start(f any) error {
+func Start(f Handler) error {
```

Comment:
I changed this - since we _want_ a proper Handler to be passed here. 

We shouldn't be failing this during runtime - it will swallow errors.

Reply:
Well, I thought so to.  But then thought it would be possible one might want a Function which fires "OnStart" and "OnStop" but does not accept HTTP requests, or is set to never scale to 0 and performs some action on a chron-like timer.   This would open us up to having Functions which fill very different roles than the expected, request-response HTTP cycle... and I think that's a good thing.  WDYT?

Reply:
> This would open us up to having Functions which fill very different roles than the expected, request-response HTTP cycle

For these use cases I'd use a different template and not the HTTP one

Reply:
Secondly, if the user typos `Handle` then they won't get feedback here because it's loosely typed.